editor/debugger/editor_performance_profiler.cpp

```diff
@@ -390,16 +399,16 @@ void EditorPerformanceProfiler::_notification(int p_what) {
 }
 
 EditorPerformanceProfiler::EditorPerformanceProfiler() {
-	set_name(TTR("Monitors"));
+	set_name(TTRC("Monitors"));
 	set_split_offset(340 * EDSCALE);
 
 	monitor_tree = memnew(Tree);
 	monitor_tree->set_custom_minimum_size(Size2(300, 0) * EDSCALE);
 	monitor_tree->set_auto_translate_mode(AUTO_TRANSLATE_MODE_DISABLED);
 	monitor_tree->set_columns(2);
-	monitor_tree->set_column_title(0, TTR("Monitor"));
+	monitor_tree->set_column_title(0, TTRC("Monitor"));
 	monitor_tree->set_column_expand(0, true);
-	monitor_tree->set_column_title(1, TTR("Value"));
+	monitor_tree->set_column_title(1, TTRC("Value"));
```

Comment:
`monitor_tree` has auto translation disabled and column title does not have a separate auto translate mode.

We should probably make auto translation enabled for the `Tree` and disabled for each `TreeItem`.